### ISSUE TITLE
Clearly support more modern JQuery library in kalatheme

### ISSUE THREAD

**Issue by labboy0276** [drupalprojects/kalatheme#197]
I think we should make a string effort to get Kalatheme to work with some of the latest JQuery Version without having to jump through bells and whistles.

We should also include support for JQuery 2.x.  Right now when yo switch to a 2.x branch, Kalatheme says no thanks.

Anywho, Right now to get Kalatheme to work with say 1.10, you have to use JQuery Update 7.x-3.0-alpha2.  Turn on JQuery Migrate and select 1.10.  This is needed since Bootstrap 3.3+ and people dont know to do this.

**Comment by RobLoach** [comment]
jQuery Update 3.x is still in alpha. Should jQuery Update default to using the latest 1.x, and use jQuery Migrate?

**Comment by labboy0276** [comment]
Yes, we should at least get this stable on 1.10 / 1.11 and the latest bootstrap 3.3.5 with no weird sadness. @RobLoach
